nasa/cumulus
# CUMULUS-3449:Update cumulus_id type add indexes

**Summary:** Summary of changes

Addresses [CUMULUS-3449: SPIKE: Improve performance for reconciliation query](https://bugs.earthdata.nasa.gov/browse/CUMULUS-3449)

## Changes

* Update Cumulus_id Type and Indexes
* Add migration instruction

## Test Instructions:

1. Build and deploy your stack from master branch.
2. Add a few granules with modified spec/parallel/ingestGranule/IngestGranuleSuccessSpec.js.  Comment out afterAll and skip the test steps:
xdescribe('when moving a granule'
xit('can delete the ingested granule from the API'
3. Follow migration instruction to update cumulus_id type and indexes. Verify that type and indexes have been updated.
4. Build from PR branch.  Update example/data-persistence-tf/main.tf, set dbRecreation=false.  Deploy data-persistence-tf module. Verify that granules_collection_cumulus_id_granule_id_unique UNIQUE CONSTRAINT is added and old constraint is dropped.


## PR Checklist
